source/components/detectors.md

```diff
@@ -0,0 +1,48 @@
+# Detectors
+
+The following table summarizes all detectors used in Vijil. The module name is how the detector will appear in some logs.
+
+| Detector                             | Module name                                                      | Description                                                                                                                                                                                                            |
```

Comment:
i don't think we're surfacing the module names to the end user. so we can delete that column?

Reply:
will they also be purged from the logs?

Reply:
ah, good point. no i think we should keep them in the logs.

where i was coming from is the preference of keeping the documentation itself clean of any IDs used in the backend. maybe a page linked from here that lists the name to ID mapping of all entities?

Reply:
removed them; will add other page in diff PR